package.json

```diff
@@ -82,6 +82,10 @@
       "lcov",
       "text",
       "html"
+    ],
+    "exclude": [
```

Comment:
This stuff is actually in master right now, not sure why it's still showing up in this PR. Ignore it :-)